mne/defaults.py

```diff
@@ -10,7 +10,7 @@
     color=dict(mag='darkblue', grad='b', eeg='k', eog='k', ecg='m', emg='k',
                ref_meg='steelblue', misc='k', stim='k', resp='k', chpi='k',
                exci='k', ias='k', syst='k', seeg='saddlebrown', dipole='k',
-               gof='k', bio='k', ecog='k', hbo='darkblue', hbr='b',
+               gof='k', bio='k', ecog='k', hbo='r', hbr='b',
```

Comment:
is there a chance that this is confounded by red which means bad epoch in epochs.plot or evoked.plot when you show bads?

Reply:
Sounds like that's going to be a problem! (I haven't got to evoked.plot yet).

I was just trying to bring MNE inline with convention for fNIRS. My goal over the next few weeks is to get to a plot that looks like this...

![image](https://user-images.githubusercontent.com/748691/67904571-dbb7e500-fbc2-11e9-9ba5-4b2d75c48e11.png)

The plan was to figure out how to make default colors match convention, then figure out how to plot two different types on the same figure. But sounds like changing the raw traces to red might be a bad idea. Would a different red be possible? If not, lets leave as is and drop this PR.

Reply:
> Would a different red be possible? If not, lets leave as is and drop this PR.

Yes I think some other shade of red could work. Maybe a dark red or orangeish or purpelish red? Maybe `#AA3377`? I often look at [this PDF](https://personal.sron.nl/~pault/data/colourschemes.pdf) for color ideas

Reply:
> I was just trying to bring MNE inline with convention for fNIRS.

... as I think we should try to follow accepted convention as much as possible. So if we can get something "red enough" to not surprise and/or annoy fNIRS folks and also something sufficiently "not red" that our bad channel marking (which is effectively `#FF0000` IIRC) still stands out we should be good.